presto-main/src/main/java/io/prestosql/operator/PipelineStats.java

```diff
@@ -344,18 +344,33 @@ public boolean isFullyBlocked()
         return blockedReasons;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the physical input data size in bytes.
+     *
+     * @return the size of the input data in bytes.
+     */
     @JsonProperty
     public DataSize getPhysicalInputDataSize()
     {
         return physicalInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the physical input positions of the PipelineStats.
```

Comment:
It's not clear to me what is this "physical input positions". Is any other way to make it clear for the reader?